pkg/model/project.pb.go

```diff
@@ -305,6 +305,15 @@ func (x *Project) GetStaticAdminDisabled() bool {
 	return false
 }
 
+func (x *Project) GetStaticAdminUsername() string {
```

Comment:
This file is generated by protoc-gen-go, so please move this method to `project.go` in the same directory.